src/index.html

```diff
@@ -11,6 +11,46 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="search-bar" data-qa="big">
+      <form
+        action="#"
+        method="get"
+        class="search-bar__form"
+      >
+        <div class="search-bar__container">
+          <div class="search-bar__content">
+            <input
+              type="text"
+              name="bigSearch"
+              id="bigSearch"
+              class="search-bar__input"
+              data-qa="keypress"
+              placeholder="Try &quot;Los Angeles&quot;"
+            >
+          </div>
+        </div>
+      </form>
+    </div>
+
+    <div class="search-bar" data-qa="small">
+      <form
+        action="#"
+        method="get"
+        class="search-bar__form"
+      >
+        <div class="search-bar__container search-bar__container--small">
+          <div class="search-bar__content search-bar__content--small">
+            <input
+              type="text"
+              name="smallSearch"
+              id="smallSearch"
+              class="search-bar__input search-bar__input--small"
+              placeholder="Try &quot;Los Angeles&quot;"
+            >
+          </div>
+        </div>
+      </form>
+    </div>
+
   </body>
```

Comment:
```suggestion
    </div>
  </body>
```
Don't leave empty lines between parent and child elements